### PR TITLE
Migrations scripts for Armadillo 2.x -> 3.0

### PR DESCRIPTION
**Pull request by tommydeboer**

### CODE REVIEW

File: scripts/migrate-auth.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+    # 1. Set up clients
+    armadillo_client = create_armadillo_client()
+    fusion_client = create_fusion_client()
+
+    # 2. Get production apps that have users
+    users = get_active_registered_users(fusion_client)
+    apps = get_production_applications_with_users(fusion_client, users)
+
+    # 3. Let user pick which app to migrate to this Armadillo server
+    app_to_migrate = choose_application(apps)
```
Comment (mswertz) [inline scripts/migrate-auth.py:63]: Did you test this with a 'data manager' user? I.e. what permissions must a user have to do this? Data managers have 'SU' in only one or more applications, would they also be able to do this?

Comment (mswertz) [reply scripts/migrate-auth.py:63]: Oh, and what happens when user doesn't have any applications to choose from?

**Review by mswertz — APPROVED** [review]
okay, lovely code. Just two small questions you might one to check before merge.

File: scripts/migrate-auth.py (hunk 1 above)
Comment (tommydeboer) [reply scripts/migrate-auth.py:63]: As discussed, we will use API key for now and if it happens that a project's SU wants to do the migration theirselves, then we will add that option later.

**Comment by tommydeboer** [comment]
[Migration script for Fusion auth [blocks on cluster]](https://trello.com/c/nG3erycy/945-migration-script-for-fusion-auth-blocks-on-cluster)

**Pull request merged**